src/randoop/main/GenInputsAbstract.java

```diff
@@ -409,17 +373,12 @@ public GenInputsAbstract(String command, String pitch,
   public static boolean dont_output_tests = false;
   
   /**
-   * Output sequences even if they do not complete execution.
-   * 
-   *  Randoop's default behavior is to output only tests consisting of
-   *  method call sequences that execute every statement, rather than throwing
-   *  an exception or failing a contract check before the last statement.
+   * Indicate which classes that any generated test must include. Tests that
+   * do not have classes matching the regular expression are not written to 
+   * the test suites. 
    */
-  @Option("Output sequences even if they do not complete execution")
-  public static boolean output_nonexec = false;
-  
-  @Option("specifies regex of classes that must be in any regression tests")
-  public static Pattern test_classes = null;
+  @Option("Regular expression for class names that must occur in generated tests")
```

Comment:
Looking at the code, it's not just that the generated test must include the given class names; if any variable has any class name that isn't in this list, the test is discarded.  That is, the condition seems stronger than stated here.  Can you please double-check and update the documentation?